Packs/AzureStorageTable/Integrations/AzureStorageTable/AzureStorageTable_test.py

```diff
@@ -0,0 +1,277 @@
+from CommonServerPython import *
+
+ACCOUNT_NAME = "test"
+BASE_URL = f'https://{ACCOUNT_NAME}.table.core.windows.net/'
+SAS_TOKEN = "XXXX"
+API_VERSION = "2020-10-02"
+
+
+def load_mock_response(file_name: str) -> str:
+    """
+    Load mock file that simulates an API response.
+    Args:
+        file_name (str): Name of the mock response JSON file to return.
+    Returns:
+        str: Mock file content.
+    """
+    with open(f'test_data/{file_name}', mode='r', encoding='utf-8') as mock_file:
+        return mock_file.read()
+
+
+def test_azure_storage_create_table_command(requests_mock):
+    """
+    Scenario: Create Table.
+    Given:
+     - User has provided valid credentials.
+    When:
+     - azure-storage-table-table-create called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure a sample value from the API matches what is generated in the context.
+    """
+    from AzureStorageTable import Client, create_table_command
+
+    mock_response = json.loads(load_mock_response('create_table.json'))
+    url = f'{BASE_URL}Tables{SAS_TOKEN}'
+    table_name = 'test'
+
+    requests_mock.post(url, json=mock_response)
+
+    client = Client(server_url=BASE_URL, verify=False, proxy=False,
+                    account_sas_token=SAS_TOKEN,
+                    storage_account_name=ACCOUNT_NAME, api_version=API_VERSION)
+
+    result = create_table_command(client, {'table_name': table_name})
+
+    assert len(result.outputs) == 1
+    assert result.outputs_prefix == 'AzureStorageTable.Table'
+    assert result.outputs.get('TableName') == 'xsoar'
+
+
+def test_azure_storage_delete_table_command(requests_mock):
+    """
+    Scenario: Delete Table.
+    Given:
+     - User has provided valid credentials.
+    When:
+     - azure-storage-table-table-delete called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure a sample value from the API matches what is generated in the context.
+    """
+    from AzureStorageTable import Client, delete_table_command
+
+    table_name = 'test'
+    url = f'{BASE_URL}Tables(\'{table_name}\'){SAS_TOKEN}'
+
+    requests_mock.delete(url, text='')
+
+    client = Client(server_url=BASE_URL, verify=False, proxy=False,
+                    account_sas_token=SAS_TOKEN,
+                    storage_account_name=ACCOUNT_NAME, api_version=API_VERSION)
+
+    result = delete_table_command(client, {'table_name': table_name})
+
+    assert result.outputs is None
+    assert result.outputs_prefix is None
```

Comment:
Same as above.
Not adding more notes on this file for now, please fix accordingly